typings.d.ts

```diff
@@ -552,7 +552,7 @@ declare module 'launchdarkly-js-sdk-common' {
 
   /**
    * Describes the reason that a flag evaluation produced a particular value. This is
-   * part of the {@link LDEvaluationDetail} object returned by {@link LDClient.variationDetail]].
```

Comment:
Noticed this when working on the docs for node-client-sdk. So a good chance to just fix that link.